src/java/install.sh

```diff
@@ -142,8 +239,19 @@ if [ "${architecture}" != "amd64" ] && [ "${architecture}" != "x86_64" ] && [ "$
     exit 1
 fi
 
-# Install dependencies
-check_packages curl ca-certificates zip unzip sed
+# Install dependencies,
+check_packages ca-certificates zip unzip sed findutils util-linux tar
```

Comment:
> findutils util-linux tar

Curious, why do we need to additionally install ^ ?

Reply:
Some RHEL distributions do not install the "findutils", "util-linux", and "tar" packages by default. Mariner linux, for example does not install "util-linux" or "tar". Rocky linux minimal distributions do not install "findutils" or "tar" by default, and Alma linux minimal distributions do not install any of these packages by default.

These packages provide utilities that are needed to for "install.sh" to work.

Reply:
Makes sense, thanks for the explanation.